estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_real_estate_platform,estate_property,model_estate_property,base.group_user,1,1,1,1
+access_real_estate_type,estate_property_type,model_estate_property_type,base.group_user,1,1,1,1
+access_real_estate_tag,estate_property_tag,model_estate_property_tag,base.group_user,1,1,1,1
+access_real_estate_offer,estate_property_offer,model_estate_property_offer,base.group_user,1,1,1,1
```

Comment:
```suggestion
access_real_estate_offer,estate_property_offer,model_estate_property_offer,base.group_user,1,1,1,1

```
missing new line at the end of the file.
this goes for most of your other non-python files as well